articles/140_topic_and_service_name_mapping.md

```diff
@@ -76,14 +73,15 @@ The content of substitutions:
 
 - must not be empty
 
+- must be not longer than 256 characters
```

Comment:
This change still doesn't look right to me:

https://github.com/ros2/design/commit/bb4b2f4#diff-51ea0dd0254ef288483a09be8b28379dR77